### PR TITLE
Leaderboard view

### PR DESCRIPTION
**Pull request by reemsalous**

### CODE REVIEW

File: public/gameslist_websocket_connection.js [hunk 1]
```diff
@@ -15,23 +15,3 @@ socket.on('gamesUpdated', function(data){
             + data[i].playerCount + " players" + '</a>';}
     });
 
-function searchFunction() {
```
Comment (JonasBGerth) [inline public/gameslist_websocket_connection.js]: Should we keep this? @oscheller1

File: routes/play.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+var router = express.Router();
+var request = require('request');
+
+
+router.get('/', function (req, res, next) {
+
+
+
+
+    res.render('play', { title: req.baseUrl });
```
Comment (oscheller1) [inline routes/play.js]: Whats's this? What do we need this for?

Comment (reemsalous) [reply routes/play.js]: you can ignore it, its old one and I forgot to delete it.

Comment (oscheller1) [reply routes/play.js]: If no longer needed, please remove from code :)

**Pull request merged**